packages/hooks/src/useHasWallet.ts

```diff
@@ -0,0 +1,9 @@
+import useWalletsList from './useWalletsList';
+
+const useHasWallet = () => {
```

Comment:
Question: Do we need this extra hook when we can just a pass a boolean in `useWalletsList`

Reply:
@heorhi-deriv I think even I have the same question 😅 .

Reply:
@thisyahlen-deriv nice suggestion, we decided to apply it

Reply:
@heorhi-deriv if you think that we can expand on this in the future, then lets keep this. Since `useWalletslist` is getting more larger

Reply:
@thisyahlen-deriv as you guys have 50/50 opinions, i decided to move `has_wallet` check inside `useWalletsList`. IMHO there is a close relation between `has_wallet` and wallets itself. so i prefer to hide it inside useWalletsList